source/main/ipc/index.ts

```diff
@@ -21,8 +21,10 @@ import { handleAddressIntrospectionRequests } from './introspect-address';
 import { handleManageAppUpdateRequests } from './manageAppUpdateChannel';
 import { openExternalUrlChannel } from './open-external-url';
 import { openLocalDirectoryChannel } from './open-local-directory';
+import usbDetect from 'usb-detection';
 
 export default (window: BrowserWindow) => {
+  usbDetect.startMonitoring();
```

Comment:
This definitely needs removing – I just wanted to trigger the loading of that native library